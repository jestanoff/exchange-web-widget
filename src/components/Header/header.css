.header {
  margin: 24px 0;
}

.heading1 {
  font-size: 24px;
  letter-spacing: -1px;
  margin: 0;
  text-align: center;
}
