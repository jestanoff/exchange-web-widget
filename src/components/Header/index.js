import React from 'react';

const Header = () => (
  <header>
    <h1>Exchange</h1>
  </header>
);

export default Header;
