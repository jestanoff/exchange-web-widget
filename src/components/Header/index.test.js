import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Header from './';


describe('Header component', () => {
  test('should render proper markup', () => {
    const { container } = render(<Header />);
  
    expect(container).toMatchSnapshot();
  })
});
