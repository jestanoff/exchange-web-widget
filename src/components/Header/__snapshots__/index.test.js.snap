// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render proper markup 1`] = `
<div>
  <header>
    <h1>
      Exchange
    </h1>
  </header>
</div>
`;
